### PR TITLE
Errcheck issues fixed in server/channels/api4/brand_test.go

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28351
Jira https://mattermost.atlassian.net/browse/MM-60708
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/channels/api4/brand_test.go [hunk 1]
```diff
@@ -77,7 +79,8 @@ func TestDeleteBrandImage(t *testing.T) {
 	require.Error(t, err)
 	CheckForbiddenStatus(t, resp)
 
-	th.Client.Logout(context.Background())
+	_, err = th.Client.Logout(context.Background())
+	require.Error(t, err)
```
Comment (hanzei) [inline server/channels/api4/brand_test.go:83]: Why should this error?

**Review by hanzei — CHANGES_REQUESTED** [review]
One question

File: server/channels/api4/brand_test.go (hunk 1 above)
Comment (Aryakoste) [reply server/channels/api4/brand_test.go]: I changed this. It should be NoError

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@Aryakoste Heads up that I opened more tickets: https://github.com/mattermost/mattermost/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+errcheck+label%3A%22Up+For+Grabs%22

Are you interested in working on another `errcheck` issue?

**Comment by Aryakoste** [comment]
Yes thanks for informing I would love to work on more such issues.

**Comment by hanzei** [comment]
Awesome, I'm looking forward to more contributions from you! :rocket:

**Pull request merged**